server/src/main/java/org/elasticsearch/search/lookup/LeafDocLookup.java

```diff
@@ -18,6 +18,7 @@
 import org.elasticsearch.script.field.Field;
 
 import java.io.IOException;
+import java.lang.reflect.AccessibleObject;
```

Comment:
Is this `AccessibleObject` import necessary?

Reply:
Nope, accidentally imported, removed now